### DISCUSSION TITLE
Making the background of Form Card transparent

### DISCUSSION THREAD

**Discussion by n-srinidhi** [h2oai/wave#1775] [Q&A] [answered]
Hello!I am trying to create a page using the form_card element. I noticed that using the cards with a dark background makes it look like this:

![image](https://user-images.githubusercontent.com/117831190/210200915-b56af2f3-7057-4665-8c64-3a38caacd2e8.png)

1. The form card background colour and the colour of the background outside the form card look different. Is there a way to make the background of the cards transparent, Something like this?

![image](https://user-images.githubusercontent.com/117831190/210201015-945c4725-5671-4abc-af21-132ea3bd1dde.png)

2. Can I remove the borders of the textbox?

**Comment by mturoci** [comment, marked as answer]
Wave does not provide this level of control, but both of these can be solved using [custom CSS](https://wave.h2o.ai/docs/custom-css/).

Note that we do not recommend using it much since your style selectors can break in the next Wave version.

**Reply by n-srinidhi** [reply]
```
meta_login=ui.meta_card(
box='',
title='Login',
layouts=[
    ui.layout(
        breakpoint='xs',
        zones=[
            ui.zone(name='header'),
            ui.zone(name='top', direction=ui.ZoneDirection.ROW, size='200px'),
            ui.zone(name='main_login',align='center',justify='center')
        ]
        )
    ],
theme='neon'
)

main_login=ui.form_card(
    box='main_login',
    items=[
        ui.textbox(
            name='uid',
            placeholder='Enter User ID',
            required=True
        ),
        ui.textbox(
            name='pwd',
            placeholder='Enter Password',
            password=True,
            required=True
        ),
        ui.buttons(
            justify='center',
            items=[
            ui.button(
            name='btn_login',
            label='Sign in',
            path='',
            primary=True
            )
            ]
        )
    ]

)
```
This is how I'm implementing it now. Is it possible to fill the form card in the whole page and only customize the text boxes and button to stay in the center?This will be close to what I'm trying to achieve.

**Reply by mturoci** [reply]
> to fill the form card in the whole page

Yes, that's doable.

> the text boxes and button to stay in the center

This is not currently easily possible, you would need to use custom CSS in the meanwhile. Filed a feature request - https://github.com/h2oai/wave/issues/1780

**Reply by n-srinidhi** [reply]
Thank you!
